### PR TITLE
regenerate json file for lcid

### PR DESCRIPTION
**Pull request by natesheehan**
Regenerated JSON file for active LCID - @dabreegster can you confirm this is looking okay?

### CODE REVIEW

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Heads-up @natesheehan I don't think this is ready to go yet, it's still using the old schema with origin associated with people not trip attributes:

image

**Comment by dabreegster** [comment]
I realized I forgot about another breaking schema change on my end that's pretty important! We need to multiply `departure` by `10000.0` now too; there's a change in A/B Street serialization to compress floating point numbers. Could we also squeeze that in the `abstr` code and the test JSON file here?

**Comment by Robinlovelace** [comment]
> Could we also squeeze that in the abstr code and the test JSON file here?

Absolutely. @natesheehan are you up for opening another issue in the repo? Should be a quick fix.

**Comment by natesheehan** [comment]
https://github.com/cyipt/actdev/issues/178

**Comment by natesheehan** [comment]
I have made the changes to the abstr-scenarios.r file - this afternoon I am going to rerun the build script for sites affected by #179

File: code/abstr-scenarios.R [hunk 1]
```diff
@@ -259,29 +259,35 @@ abs((
 # mapview::mapview(houses) + mapview::mapview(buildings_in_zones) +
 #   mapview::mapview(desire_lines) + mapview::mapview(zones_of_interest)
 
+od = desire_lines %>% rename(Walk = walk_base, Bike = cycle_base, Drive = drive_base)
```
Comment (Robinlovelace) [inline code/abstr-scenarios.R:262]: :+1:

File: code/abstr-scenarios.R [hunk 2]
```diff
@@ -290,17 +296,19 @@ abb = rbind(abc, abt)
 rows_equal = nrow(abb) == sum(desire_lines$trimode_base)
 if (!rows_equal)
   stop("Number of trips in scenario different from baseline")
-abbl = abstr::ab_json(abb)
+abbl = abstr::ab_json(abb, scenario_name = "base")
 
+od_active = desire_lines %>% rename(Walk = walk_go_active, Bike = cycle_go_active, Drive = drive_go_active)
```
Comment (Robinlovelace) [inline code/abstr-scenarios.R:301]: :+1: :+1:

**Review by Robinlovelace — APPROVED** [review]
Great work @natesheehan this looks good to me.

**Comment by dabreegster** [comment]
`scenario_base.json` imports just fine, and the departure times look correct:
Screenshot from 2021-07-11 16-08-47

**Pull request merged**